cmd/main.go

```diff
@@ -153,6 +155,11 @@ func parseConfig() (spirev1alpha1.ControllerManagerConfig, ctrl.Options, []*rege
 		ctrlConfig.ClusterDomain = clusterDomain
 	}
 
+	if ctrlConfig.ParentIDTemplate != "" {
+		parentIDTemplate, err := template.New("customParentIDTemplate").Parse(ctrlConfig.ParentIDTemplate)
```

Comment:
This will alias parentIDTemplate declared in the outer scope....
```suggestion
		parentIDTemplate, err = template.New("customParentIDTemplate").Parse(ctrlConfig.ParentIDTemplate)
```